esp/src/src-react/components/DFUWorkunits.tsx

```diff
@@ -15,6 +15,7 @@ import { Filter } from "./forms/Filter";
 import { Fields } from "./forms/Fields";
 import { ShortVerticalDivider } from "./Common";
 import { selector } from "./DojoGrid";
+import { sashaService, WsSasha } from "./Sasha";
```

Comment:
This should be the same as the line from the imports in Sasha.tsx:
`import { SashaService, WsSasha } from "@hpcc-js/comms";`